metadata/src/main/java/edu/unc/lib/dl/util/DateTimeUtil.java

```diff
@@ -43,7 +43,11 @@ public static Date parseUTCDateToDate(String utcDate) throws ParseException {
 	}
 
 	public static Date parseUTCToDate(String utcDate) throws ParseException {
-		return parseUTCToDateTime(utcDate).toDate();
+		try {
```

Comment:
Because this method wasn't actually throwing ParseException. It was passing through the IllegalArgumentException from parseUTCToDateTime(), which meant the catch-block in SetRecordDatesFilter was never getting hit.